server/config/config.go

```diff
@@ -639,6 +640,8 @@ func (man Manager) addConfigs() {
 		"How much time to wait between processing software for vulnerabilities.")
 	man.addConfigString("vulnerabilities.cpe_database_url", "",
 		"URL from which to get the latest CPE database. If empty, defaults to the official Github link.")
+	man.addConfigString("vulnerabilities.cpe_translations_url", "",
+		"URL from which to get the latest CPE translations. If empty, defaults to the official Github link.")
```

Comment:
`If empty, defaults to the official Github link.` does it?

Reply:
I copied the description from the `cpe_database_url`. I think this could be rewritten to something like "If empty, the CPE (database|translations) is downloaded from the latest release from https://github.com/fleetdm/nvd/releases"